src/NexusMods.DataModel/Games/RunGameTool.cs

```diff
@@ -107,6 +113,61 @@ public async Task Execute(Loadout loadout, ApplyPlan applyPlan, CancellationToke
         _logger.LogInformation("Finished running {Program}", program);
     }
 
+    private async Task RunThroughSteam(uint appId, CancellationToken cancellationToken)
+    {
+        await _osInterop.OpenUrl(new Uri($"steam://rungameid//{appId.ToString(CultureInfo.InvariantCulture)}"), cancellationToken);
```

Comment:
☝️ Likely Bug Here

If a user has already been running an application via Steam, a `reaper` process might already exist; thus there is risk that `WaitForProcessToStart` returns the wrong `reaper` process (one belonging to an existing application).

While it may be uncommon for a user to launch 2 games at the same time; Steam also ships regular software these days. Running Steam shipped software in background may be much more common.

Some examples that come to mind:  
- Blender (Exporting 3D Models for Testing in Custom Mods)
- VTubing Software (e.g. VTube Studio, VSeeFace, FaceRig)
- VR Software (SteamVR, etc.)